README.md

```diff
@@ -1,3 +1,48 @@
 # register-country-by-country-reporting-frontend
 
-This is a placeholder README.md for a new repository
\ No newline at end of file
+## Info
+
+This service allows a user(agent, client or organisation) to register to CBCR services.
+
+This service has a corresponding back-end service, namely register-country-by-country-reporting can be viewed [here](https://github.com/hmrc/register-country-by-country-reporting) which integrates with HOD, i.e. DES/ETMP.
+
+### Dependencies
+
+| Service          | Link                                                                   |
```

Comment:
Please add other cbc services (cbc file upload frontend,backend and stubs AND cbc register backend and stubs)

Reply:
added mentioned services.